### PR TITLE
Fix crash on devices without vibration setting

### PR DESCRIPTION
**Pull request by langerhans**
Waiting for test result before merging

### CODE REVIEW

**Comment by serpandrew** [comment]
Confirmed app no longer crashes upon opening on RP4 Pro

File: app/src/main/java/de/langerhans/odintools/main/MainViewModel.kt [hunk 1]
```diff
@@ -38,15 +39,16 @@ class MainViewModel @Inject constructor(
         executor.enableA11yService()
         executor.grantAllAppsPermission()
 
-        val isOdin2 = deviceUtils.isOdin2()
+        val deviceType = deviceUtils.getDeviceTpe()
```
Comment (miguelguzmanr) [inline app/src/main/java/de/langerhans/odintools/main/MainViewModel.kt:42]: Fix typo in function "getDeviceTpe", missing letter "y" in "Type"

File: app/src/main/java/de/langerhans/odintools/tools/DeviceUtils.kt [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+import de.langerhans.odintools.tools.DeviceType.*
 import javax.inject.Inject
 
 class DeviceUtils @Inject constructor(
     private val shellExecutor: ShellExecutor
 ) {
-    fun isOdin2() = shellExecutor.executeAsRoot("getprop ro.vendor.retro.name").map { it == "Q9" }
-        .getOrDefault(false)
+
+    fun getDeviceTpe(): DeviceType {
```
Comment (miguelguzmanr) [inline app/src/main/java/de/langerhans/odintools/tools/DeviceUtils.kt:10]: Fix typo in function "getDeviceTpe", missing letter "y" in "Type"

**Review by miguelguzmanr — APPROVED** [review]
Just a small typo. Up to you to fix it or leave it. Everything else looks good. Thank you!

**Comment by langerhans** [comment]
Typos fixed, thanks. LGTM

**Pull request merged**